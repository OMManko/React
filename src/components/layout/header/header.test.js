import { shallow } from 'enzyme';
import React from 'react';
import Header from '../header/header';
import AppLogo from "../../shared/appLogo/appLogo";

let component;

describe('<Footer />', () => {
    beforeEach(() => {
        component = shallow(<Header/>);
    });
    it('should render header and match snapshot', () => {
        expect(component).toMatchSnapshot();
    });

    it('should contain logo', () => {
        expect(component).toMatchSnapshot();
        expect(component.contains(<AppLogo/>)).toBe(true);
    });

    it('should call function when button is clicked', () => {
        const instance = component.instance();
        instance.handleSearchFormOpen(true);
        expect(console.log("Open search form"));
    });
});
