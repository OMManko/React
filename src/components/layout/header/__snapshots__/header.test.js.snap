// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should contain logo 1`] = `ShallowWrapper {}`;

exports[`<Footer /> should render header and match snapshot 1`] = `ShallowWrapper {}`;
